vcs/gogpm_test.go

```diff
@@ -14,7 +15,7 @@ func TestPresentPackageRepoSimpleGopath(t *testing.T) {
 		repo, _ := PackageForImportPath("github.com/fake/library")
 		repo.ctx = build.Context{Compiler: "gc", GOPATH: gopath}
 
-		expected := path.Join(gopath, "src", "/github.com/fake/library")
+		expected := path.Join(gopath, "src", "github.com", "fake", "library")
```

Comment:
I'm not sure if you should be using `path` or `filepath` package here. It's mostly because `Dir` is not documented, so I can't tell if it returns a "slash-separated path" or a "file system path". If the former, then `path` is correct. If latter, then you should use `filepath`.